docs/conf.py

```diff
@@ -1,131 +1,70 @@
+# -- Read the Docs Specific Configuration --------------------------------------
 
+# This needs to be done before sunpy is imported
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 if on_rtd:
     os.environ['SUNPY_CONFIGDIR'] = '/home/docs/'
     os.environ['HOME'] = '/home/docs/'
     os.environ['LANG'] = 'C'
     os.environ['LC_ALL'] = 'C'
 
-try:
-    import zeep
-except ImportError as e:
-    raise Exception(e, 'ERROR: zeep could not be imported. Building the documentation requires '
-                    'the "zeep" package to be installed')
-
-try:
-    import skimage
-except ImportError as e:
-    raise Exception(e, 'ERROR: skimage could not be imported. Building the documentation requires '
-                    'the "scikit-image" package to be installed')
+# -- Non stdlib imports --------------------------------------------------------
 
-try:
-    import drms
-except ImportError as e:
-    raise Exception(e, 'ERROR: drms could not be imported. Building the documentation requires '
-                    'the "drms" package to be installed')
+import ruamel.yaml as yaml
+from sphinx_gallery.sorting import ExplicitOrder
+from sphinx_gallery.sorting import ExampleTitleSortKey
 
-try:
-    import glymur
-except ImportError as e:
-    raise Exception(e, 'ERROR: glymur could not be imported. Building the documentation requires '
-                    'the "glymur" package to be installed')
+import sunpy
+from sunpy import __version__
 
-try:
-    import sqlalchemy
-except ImportError as e:
-    raise Exception(e, 'ERROR: sqlalchemy could not be imported. Building the documentation requires '
-                    'the "sqlalchemy" package to be installed')
+# -- Project information -------------------------------------------------------
 
-try:
-    import astroquery
-except ImportError as e:
-    raise Exception(e, 'ERROR: astroquery could not be imported. Building the documentation requires '
-                    'the "astroquery" package to be installed')
+project = 'SunPy'
+author = 'The SunPy Community'
+copyright = '{}, {}'.format(datetime.datetime.now().year, author)
 
-try:
-    import jplephem
-except ImportError as e:
-    raise Exception(e, 'ERROR: jplephem could not be imported. Building the documentation requires '
-                    'the "jplephem" package to be installed')
-
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-# The short X.Y version.
-from sunpy import __version__
-version = '.'.join(__version__.split('.')[:3])
-# The full version, including alpha/beta/rc tags.
+# The full version, including alpha/beta/rc tags
 release = __version__
-# Is this version a development release
-is_development = '.dev' in release
+is_development = '.dev' in __version__
 
-# -- Shut up numpy warnings from WCSAxes --------------------------------------
-import numpy as np
-np.seterr(invalid='ignore')
```

Comment:
+5 points for getting rid of this

Reply:
Looks like I might need to add it back, getting loads of:

```
/home/stuart/.virtualenvs/sunpy-dev/lib/python3.8/site-packages/matplotlib/axes/_base.py:3270: DeprecationWarning: In future, it will be an error for 'np.bool_' scalars to be interpreted as an index
  left, right = sorted([left, right], reverse=reverse)
```

Reply:
that's a mpl warning though, so clearly don't need to add this back :facepalm:

Reply:
The answer is not to put it back, the answer is always to fix the warnings 😉

Reply:
go put your mpl hat on :stuck_out_tongue_winking_eye: